### PR TITLE
gungnir: init at 1.0.8

### PR DESCRIPTION
**Pull request by CherryKitten**
## Description of changes

https://github.com/g0ldencybersec/gungnir is a command-line tool written in Go that continuously monitors certificate transparency (CT) logs for newly issued SSL/TLS certificates. 

## Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by CherryKitten** [comment]
> you should leave the `v` in front of the version in the commit message out. that how people do it if you `git log --oneline`.
> 
> https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#commit-conventions says `init at version` and i believe the `v` is not actually part of the version but makes it clear that the following text is a version.

thanks, I've updated the commit message

File: pkgs/by-name/gu/gungnir/package.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    rev = "v${version}";
+    hash = "sha256-k6fxAvUBAAcTHzdeGhekYhPpnS05jHq/7EqxafQfMio=";
+  };
+
+  vendorHash = "sha256-r2aU59L0fnSdc/lpR04K/GQ1eZ7ihV+tKlyuS6sPX2o=";
+
+  ldflags = [ "-s" "-w" ];
+
+  meta = with lib; {
+    description = "command-line tool that continuously monitors certificate transparency (CT) logs for newly issued SSL/TLS certificates.";
```
Comment (auroraanna) [inline pkgs/by-name/gu/gungnir/package.nix:22]: according to <https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes> you're missing capitalization and no period at the end
```suggestion
    description = "A command-line tool that continuously monitors certificate transparency (CT) logs for newly issued SSL/TLS certificates";
```

**Comment by auroraanna** [comment]
it goes so fast damn

**Review by auroraanna — CHANGES_REQUESTED** [review]

File: pkgs/by-name/gu/gungnir/package.nix (hunk 1 above)
Comment (CherryKitten) [reply pkgs/by-name/gu/gungnir/package.nix]: Updated, thanks! Seems like I overlooked the part with the conventions for the meta attributes ^^

**Review by auroraanna — APPROVED** [review]

**Comment by RaghavSood** [comment]
Result of `nixpkgs-review pr 311881` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>gungnir</li>
  </ul>
</details>

**Review by RaghavSood — APPROVED** [review]

**Pull request merged**